src/handlers/flow/FlowStreamHandler.h

```diff
@@ -57,7 +72,15 @@ struct DeviceEnrich {
     std::unordered_map<uint32_t, InterfaceEnrich> interfaces;
 };
 
-typedef std::unordered_map<std::string, DeviceEnrich> EnrichMap;
+struct SummaryData {
+    IpSummary type{IpSummary::None};
+    std::vector<Ipv4Subnet> ipv4_exclude_summary;
```

Comment:
Do we have a sense of the memory impact for these?

Reply:
Those are vector with low data points like two and three.

And this structure is kept in MetricsManager and buckets only has a pointer reference to it